src/cocoa/toga_cocoa/store.py

```diff
@@ -0,0 +1,23 @@
+from .libs import *
+
+
+class Store:
+    def __init__(self, interface):
+        self.interface = interface
+        self.interface._impl = self
+        self.app_id = self.interface.app.app_id
+        print('app_id', self.app_id)
+        # self.native = NSUserDefaults.initWithSuiteName = self.app_id
+        self.native = NSUserDefaults.standardUserDefaults
```

Comment:
Isn't this problematic from a cardinality point of view? This API would allow an end user to define multiple Store objects, but the underlying API would only allow a single instance (because the backing store is a singleton).